seed/analysis_pipelines/better/client.py

```diff
@@ -19,6 +19,24 @@ class BETTERClient:
     def __init__(self, token):
         self._token = f'Token {token}'
 
+    def test_token(self):
```

Comment:
This change is unrelated to the meter stuff -- we were using the /buildings/ api to check if tokens were valid when creating the analysis but as we created buildings this endpoint got much too slow. We should ask BETTER team to create a token endpoint for testing. For now we just use their weathers API and check if we got a 401